google-smart-home/tests/json/query/response.json

```diff
@@ -13,8 +13,7 @@
         "status": "SUCCESS",
         "brightness": 80,
         "color": {
-          "name": "cerulean",
-          "spectrumRGB": 31655
+          "spectrumRgb": 31655
```

Comment:
are you sure about this? i took the JSON from the docs
https://developers.google.com/assistant/smarthome/develop/process-intents#handle_query_intents

Reply:
Well that's weird, the docs are inconsistent. I was following https://developers.google.com/assistant/smarthome/traits/colorsetting#device-states, but https://developers.google.com/assistant/smarthome/reference/intent/query#response matches what you had before, which is the same as the format for execute commands. The former even has a note at the end about the different capitalisation between `spectrumRGB` for the command and `spectrumRgb` for the state, so I think it's probably the correct one. I'll file a bug against the documentation though.